packages/side-nav/test/side-nav-item.test.js

```diff
@@ -94,6 +94,41 @@ describe('side-nav-item', () => {
     });
   });
 
+  describe('path alias', () => {
+    beforeEach(async () => {
+      item = fixtureSync(`<vaadin-side-nav-item path="/path" path-aliases="/"></vaadin-side-nav-item>`);
+      await nextRender();
+    });
+
+    it('should set active property to true for matching alias with path', () => {
+      expect(item.active).to.be.true;
+    });
+
+    it('should set active property to false for matching alias without path', async () => {
+      item.path = undefined;
+      await item.updateComplete;
+      expect(item.active).to.be.false;
+    });
+
+    it('should update active property when aliases are updated', async () => {
+      item.pathAliases = 'aliases';
+      await item.updateComplete;
+      expect(item.active).to.be.false;
+    });
+
+    it('should set active property to true for matching empty alias', async () => {
+      item.pathAliases = '';
+      await item.updateComplete;
+      expect(item.active).to.be.true;
+    });
+
+    it('should set active property to true for a match in multiple aliases', async () => {
+      item.pathAliases = '/,/alias';
```

Comment:
**nit:** It might be a good idea to swap `/` and `/alias` to additionally ensure that aliases are correctly taken into account regardless of their position.

```suggestion
      item.pathAliases = '/alias,/';
```